### PR TITLE
Users/ajya/azurewebapppowershellversion support

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/task.json]: Patch version?

File: Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1 [hunk 2]
```diff
@@ -28,7 +28,10 @@ param
     $TakeAppOfflineFlag,
 
     [String] [Parameter(Mandatory = $false)]
-    $VirtualApplication
+    $VirtualApplication,
+
+	[String] [Parameter(Mandatory = $false)]
+	[String] $AdditionalArguments
```
Comment (pavanadepu2) [inline Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1]: Log the AdditionalArguments

File: Tasks/AzureRmWebAppDeployment/task.json (hunk 1 above)
Comment (pavanadepu2) [inline Tasks/AzureRmWebAppDeployment/task.json]: +1

File: Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1 [hunk 3]
```diff
@@ -52,6 +55,12 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 Import-Module ./AzureUtility.ps1 -Force
 Import-Module ./Utility.ps1 -Force
 
+ # Importing required version of azure cmdlets according to azureps installed on machine
+ $azureUtility = Get-AzureUtility
+
+ Write-Verbose -Verbose "Loading $azureUtility"
```
Comment (anaggar) [inline Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1]: remove -verbose statements please from everywhere if applicable

File: Tasks/AzureRmWebAppDeployment/AzureUtilityLTE9.8.ps1 [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+function Get-AzureRMWebAppProfileForMSDeployWithProductionSlot
+{
+    param([String][Parameter(Mandatory=$true)] $webAppName,
+          [String][Parameter(Mandatory=$true)] $resourceGroupName)
+
+    Switch-AzureMode AzureResourceManager
+
+    Write-Verbose "`t [Azure Call]Getting publish profile file for azureRM WebApp:'$webAppName' under Production Slot."
```
Comment (anaggar) [inline Tasks/AzureRmWebAppDeployment/AzureUtilityLTE9.8.ps1]: somewhere we have used [AzureRM Call] and some where [Azure Call] please make it consistent use azure call to be generic w.r.t azure platform

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 5]
```diff
@@ -107,6 +107,14 @@
...
             "required": false,
             "helpMarkDown": "Specify the name of the Virtual Application that has been configured in the Azure portal. The option is not required for deployments to the website root."
+         },
+        {
+            "name": "AdditionalArguments",
+            "type": "string",
+            "label": "Additional Arguments",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Additional MSDeploy.exe arguments that will be applied while uploading asp.net website."
```
Comment (anaggar) [inline Tasks/AzureRmWebAppDeployment/task.json]: get helpmarkdown message reviewed by Ambrish please

**Pull request merged**